pykube/objects.py

```diff
@@ -131,18 +131,24 @@ def watch(self):
             "metadata.name": self.name
         }).watch()
 
-    def update(self):
+    def patch(self, patch):
         '''
-        Update the Kubernetes resource by calling the API (patch)
+        Patch the Kubernetes resource by calling the API.
```

Comment:
`kubectl patch` supports multiple patch formats. I recommend documenting that this method only accepts what `kubectl patch` calls a _strategic merge patch_.
```suggestion
        Patch the Kubernetes resource by calling the API with a "strategic merge" patch.
```